inspire/base/searchext/mappings/jobshidden.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Yep, currently it's simply based on a flag, controlled by a cataloguer:

980__a:JOB Vs. 980__a:JOBHIDDEN

I assume it will still stay like that, i.e. a curator will actively flip the switch. Maybe we can consider adding a regular task to alert cataloguers about expired job positions.